_includes/slideout-menu.html

```diff
@@ -1,21 +1,29 @@
 <!-- Slideout menu -->
 <script src="https://cdnjs.cloudflare.com/ajax/libs/slideout/1.0.1/slideout.min.js"></script>
 <script>
-  var slideout = new Slideout({
-    'panel': document.getElementById('panel'),
-    'menu': document.getElementById('menu'),
-    'padding': 241,
-    'tolerance': 70
-  });
+  (function () {
+    // NOTE: if the menu element does not exist, do not initialize slideout
+    var menu = document.getElementById('menu');
+    if (!menu) {
+      return;
+    }
 
-  // Toggle button
-  document.querySelector('.toggle-button').addEventListener('click', function() {
-    slideout.toggle();
-  });
+    var slideout = new Slideout({
+      'panel': document.getElementById('panel'),
+      'menu': menu,
+      'padding': 241,
+      'tolerance': 70
+    });
 
-  $(window).resize(function() {
-    if ($(window).width() > 480) {
-      $('.main-wrapper').removeAttr("style");
-    }
-  });
-</script>
\ No newline at end of file
+    // Toggle button
+    document.querySelector('.toggle-button').addEventListener('click', function() {
+      slideout.toggle();
+    });
+
+    $(window).resize(function() {
+      if ($(window).width() > 480) {
+        $('.main-wrapper').removeAttr("style");
+      }
+    });
+  })();
+</script>
```

Comment:
These changes are to correct a javascript error that was being thrown in the book. It was more topical than anything - the error wasn't actually causing JS to stop executing - but I put this in place just in case it caused anything nastier down the road.

This fix just makes sure the element is present before trying to do anything, and it also adds some variable safety by putting the whole thing inside a closure. FE dev-speak for a 'rainy day' fix.

It does exactly what it was doing before, as long as the element is present on the page.